test/unit/src/geometries/EdgesGeometry.tests.js

```diff
@@ -285,7 +285,7 @@ export default QUnit.module( 'Geometries', () => {
 
 		QUnit.test( "three triangles, coplanar first", ( assert ) => {
 
-			testEdges( vertList, [ 0, 1, 2, 0, 2, 3, 0, 4, 2 ], 7, assert );
+			testEdges( vertList, [ 0, 2, 3, 0, 1, 2, 0, 4, 2 ], 7, assert );
```

Comment:
This is testing a case where the source geometry is composed of three triangles  that all meet at a single edge. Depending on which of the two triangles are compared at that edge first it will either be included or excluded from the final edge geometry. In one case the angles of the two triangles will meet the angle threshold and in the other it will not so the EdgeGeometry edge generation algorithm is sensitive to the order of these indices -- this new approach in this PR is just sensitive in a slightly different way. You can see in the test below this one that a different order is tested and 6 edges are returned (which is what this one returned before I changed the order).